onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -351,6 +352,9 @@ void addGlobalMethods(py::module& m, const Environment& env) {
   m.def(
       "get_device", []() -> std::string { return BACKEND_DEVICE; },
       "Return the device used to compute the prediction (CPU, MKL, ...)");
+  m.def(
```

Comment:
How does this new global seed correlate/interact with the seed recently added into ORTTrainer frontend on PR https://github.com/microsoft/onnxruntime/pull/3209?

Do we still need a seed on ORTTrainer frontend now that a global seed is being added?

Reply:
ORTTrainer seed is removed in favor of this API.